src/napari_deeplabcut/_widgets.py

```diff
@@ -632,3 +684,116 @@ def dropEvent(self, event):
         """
         self._update_property("drag", False)
         self.sig_dropped.emit(event)
+
+
+class ColorSchemeDisplay(QScrollArea):
+
+    class LabelPair(QWidget):
```

Comment:
I'd take this class out of ColorSchemeDisplay, so we could perhaps reuse it elsewhere.